tools/sdk-generation-pipeline/packages/sdk-generation-cli/src/utils/autorestConfigExtractorUtils.ts

```diff
@@ -0,0 +1,29 @@
+import * as fs from 'fs';
+import * as os from 'os';
+import * as path from 'path';
+import winston from 'winston';
+
+export function extractAutorestConfigs(autorestConfigFilePath: string, sdkRepo: string, logger: winston.Logger): string | undefined {
```

Comment:
We should try to use try to use the autorest configuration module if possible. See comments in https://github.com/Azure/azure-sdk-tools/issues/3390

Reply:
No, we don't need to parse the autorest configuration here and only want to input the unparsed autorest configuration to sdk automation script because:
1. It's easy for sdk owners to generate autorest.md by using unparsed autorest configuration.
2. The autorest configuration inputted manually(in a file or in a pr comment) is a invalided autorest configuration because we want to simpify the input. For example, the value of `require` is a invalid relative path to swagger readme.md, there will be unexpected loading error when using autorest configuraiton module.